packages/networks/bin/localhost.js

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs-extra'
+import path from 'path'
+
+// We use Partial<NetworkConfig> for localhost as we don't have all the information
+const defaultLocalhost = {
+  chain: 'localhost',
+  description: 'Localhost network.',
+  featured: false,
+  fullySubsidizedGas: true,
+  id: 31337,
+  isTestNetwork: true,
+  name: 'Localhost',
+  nativeCurrency: {
+    coingecko: 'ethereum',
+    decimals: 18,
+    name: 'ETH',
+    symbol: 'ETH',
+  },
+  provider: 'http://127.0.0.1:8545',
+  publicLockVersionToDeploy: 13,
+  publicProvider: 'http://127.0.0.1:8545',
+  subgraph: {
+    endpoint: '',
+  },
+}
+
+const generateLocalhostNetworkFile = ({
+  unlockAddress,
+  subgraphEnpoint = 'http://localhost:8000/subgraphs/name/testgraph',
+}) => {
+  const localhost = {
+    ...defaultLocalhost,
+    subgraph: {
+      endpoint: subgraphEnpoint,
+    },
+    unlockAddress,
+  }
+
+  // log for debug purposes
+  console.log(localhost)
```

Comment:
Do you wnat to keep that?

Reply:
yes lets keep that one so we have it to debug on CI